internal/bminventory/inventory.go

```diff
@@ -574,9 +573,9 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 		return nil, common.NewApiError(http.StatusBadRequest, err)
 	}
 
-	b.setDefaultUsage(&cluster.Cluster)
+	setDiskEncryptionWithDefaultValues(&cluster.Cluster, params.NewClusterParams.DiskEncryption)
 
-	setDiskEncryption(&cluster.Cluster, params.NewClusterParams.DiskEncryption)
+	b.setDefaultUsage(&cluster.Cluster)
```

Comment:
disk encryption setup should be added to setDefaultUsage

Reply:
Do you need to report always on usage or just in certain cases?

Reply:
> disk encryption setup should be added to setDefaultUsage
This is not related to the usage, but to the base [PR](https://github.com/openshift/assisted-service/pull/2643) (please ignore the first commit)

Reply:
> Do you need to report always on usage or just in certain cases?

I will rather be consistent - each time we register/update a cluster we register/update the usage.